main.tf

```diff
@@ -87,8 +87,23 @@ resource "aws_lambda_function" "logs_to_datadog" {
       DD_STORE_FAILED_EVENTS = var.store_failed_events
       DD_S3_BUCKET_NAME      = module.datadog_serverless_s3.bucket_name
       ## Filter out lambda platform logs
-      DD_LOGS_CONFIG_PROCESSING_RULES = "[{\"type\": \"exclude_at_match\", \"name\": \"exclude_start_and_end_logs\", \"pattern\": \"(START|END) RequestId\"}]"
-      EXCLUDE_AT_MATCH                = "\"(START|END) RequestId:\\s"
+      DD_LOGS_CONFIG_PROCESSING_RULES = jsonencode([
+        {
+          type    = "exclude_at_match",
+          name    = "exclude_start_and_end_lamda_platform_logs",
+          pattern = "(START|END) RequestId"
+        },
+        {
+          type    = "exclude_at_match",
+          name    = "exclude_dms_s3_access_logs",
+          pattern = "(cb6286a0d1e7cf75494d129a44503b1e5238eca143859e52c4e36251c9527208|f0f74535e58c162dd3ac99ed60ebfaf413518e6e212a5d364ce226cbed800d01|9842704febb1bf081c69ebcd73febd166602d29de15d30268d3f27ac0b0bedb8)" # The canonical ID of the AWS account that owns the bucket (dw-us-dev/qa/prod accounts)
+        },
+        {
+          type    = "exclude_at_match",
+          name    = "exclude_ota_device_logs_304",
+          pattern = "/device-ota/.*\\s304"
+        }
```

Comment:
`.*` makes me nervous in an exclusion.
This also doesn't seem to be working at least in the log search.
![Screenshot 2024-09-17 at 4 54 27 PM](https://github.com/user-attachments/assets/c8f3435c-3101-4f88-a56f-652f670fff67)

Reply:
Yes, but the `exclude_at_match` takes a regex pattern 👍